src/main/webapp/assets/modules/contests/contests.con.js

```diff
@@ -1,4 +1,4 @@
-techlooper.controller('contestsController', function (apiService, $scope, jsonValue) {
+techlooper.controller('contestsController', function (apiService, $scope, jsonValue, $window) {
```

Comment:
Line is too long.<br>'techlooper' is not defined.<br>'$window' is defined but never used.